commands/version.go

```diff
@@ -0,0 +1,41 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/fnproject/cli/client"
+	"github.com/fnproject/cli/config"
+	"github.com/urfave/cli"
+)
+
+// VersionCommand
+func VersionCommand() cli.Command {
+	return cli.Command{
+		Name:        "version",
+		Usage:       "Display CLI and server versions",
+		Description: "This is commands shows the latest client and server version.",
```

Comment:
"This command shows the version of the Fn CLI being used and the version of the Fn Server referenced by the current context, if available."